test/jdk/javax/swing/JFileChooser/FileChooserSymLinkTest.java

```diff
@@ -21,86 +21,124 @@
+ *          is valid on ValueChanged property listener.
+ * @run main/manual FileChooserSymLinkTest
+ */
 public class FileChooserSymLinkTest {
     static JFrame frame = null;
+    static Boolean bMultiSel_Enabled = false;
+    static JFileChooser jfc = null;
+    static JPanel panel = null;
+    static JTextArea textArea = null;
+    static JCheckBox checkMSelection = null;
     static PassFailJFrame passFailJFrame = null;
 
     public static void main(String[] args) throws Exception {
-        Robot robot = new Robot();
         SwingUtilities.invokeAndWait(new Runnable() {
             public void run() {
                 try {
                     initialize();
-                } catch (InterruptedException e) {
-                    throw new RuntimeException(e);
-                } catch (InvocationTargetException e) {
+                } catch (InterruptedException | InvocationTargetException e) {
                     throw new RuntimeException(e);
                 }
             }
         });
-
-        robot.delay(2000);
-        robot.waitForIdle();
-
         passFailJFrame.awaitAndCheck();
     }
 
     static void initialize() throws InterruptedException, InvocationTargetException {
+        //Initialize the components
         final String INSTRUCTIONS = """
                 Instructions to Test:
-                1. Create a regular directory in any specific path.
-                    ex: mkdir C:\\target
-                2. Create a Symbolic link targeting the created test directory.
-                    ex : mklink /D C:\\link C:\\target
-                3. In JFileChooser, navigate to "link" created directed.
-                4. On click of the "link" directory, if the Absolute path of
-                    Symbolic Link is valid then Click PASS, else if it is
-                    null then Click FAIL.(Including MultiSelection Mode SET)
+                1. Open an elevated Command Prompt.
+                2. Paste the following commands:
+                cd /d C:\
+                mkdir FileChooserTest
+                cd FileChooserTest
+                mkdir target
+                mklink /d link FileChooserTest
+
+                3. Navigate to C:\\FileChooserTest in the JFileChooser.
+                4. Use "Enable Multi-Selection" checkbox to enable/disable
+                   MultiSelection Mode
+                5. MultiSelection Mode disabled - On click of the "link"
+                   directory, if the Absolute path of Symbolic Link is
+                   valid then Click PASS, else if it is null then Click FAIL.
+                   MultiSelection Mode Enabled - If Multi selection of
+                   directories including Symbolic Link is possible along
+                   with valid Absolute path then Click PASS, else either
+                   of them failed click FAIL.
+                6. When done with testing, paste the following commands to remove the 'filechooser' directory
```

Comment:
I propose the following text:

```
                5. Single-selection:
                   Click "link" directory, the absolute path of the symbolic
                   link should be displayed. If it's null, click FAIL.
                   Click "target" directory, its absolute path should be
                   displayed.
                    
                   Enable multiple selection by clicking the checkbox.
                   Multi-selection:
                   Click "link", press Ctrl and then click "target".
                   Both should be selected and their absolute paths should be
                   displayed.
                   
                   If "link" can't be selected or if its absolute path is null,
                   click FAIL.
                   
                   If "link" can be selected in both single- and multi-selection modes,
                   click PASS.
                6. When done with testing, paste the following commands to remove
                   the 'FileChooserTest' directory:
```

I hope the instructions for testing in 5 are clearer now.